src/pages/listing/index.js

```diff
@@ -210,6 +210,9 @@ export function controller() {
 
         if(window.confirm("Remove " + data.name + "?")) {
             ref.remove().catch(console.error.bind(console));
+
+            // Can't figure out why `ref.remove()` was redirecting to "/"
```

Comment:
Trying to track this one down again before we merge a band aid, but not having a ton of luck yet. (I originally threw my hands up and gave up because I figured it was something weird in either firebase or mithril that I didn't want to go spelunking for.)